tools/MINGW-packages/mingw-w64-ffmpeg-gpl2/PKGBUILD

```diff
@@ -81,7 +80,6 @@ build() {
     --enable-libass \
     --enable-libbluray \
     --enable-libcaca \
-    --enable-libcelt \
```

Comment:
No longer provided by MSYS2 and it is highly unlikely users are actually using this codec at this point given that Opus has integrated/replaced it.